### PR TITLE
create identified_users and anonymous_users models

### PR DESCRIPTION
**Pull request by huttotw**
This creates three new models that will hopefully make it more clear what an identified user and anonymous user are. Identified users are informed from actual identify events, where anonymous users are informed by devices who have not been identified.

### CODE REVIEW

File: models/anonymous_users.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  devices.geo_city as last_geo_city,
+  devices.geo_lat_long as last_geo_lat_long,
+  devices.event_time as last_event_time,
+  devices.event_id as last_event_id,
+from {{ ref('int_devices') }} devices
+where
+  devices.device_id is not null
+  and devices.device_num_desc = 1
+  and devices.source_type != 'server' -- exclude server events, they won't have geo or device values.
+  and devices.source_type != 'invalid' -- not sure what this is, but it doesn't have any data associated
```
Comment (sgagofs) [inline models/anonymous_users.sql:20]: Huh, where did you see invalids from?

Seeing some comments about "historical placeholders" relating to partitioning. They should be rare, hopefully!

File: models/anonymous_users.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    access: public
+    config:
+      alias: "{{ var('fullstory_anonymous_users_model_name', 'anonymous_users') }}"
+      contract:
+        enforce: true
+    description: |-
+      All users who have never been identified.
+    columns:
+      - name: device_id
+        description: '{{ doc("column_last_device_id") }}'
```
Comment (sgagofs) [inline models/anonymous_users.yml:12]: Neat.

**Comment by sgagofs** [comment]
Markdown comments are neat, selects look good, documentation looks clean.

Couple thoughts:
- Would any integration tests for these be worthwhile to author?
- Yeah, let's see where invalids are coming from, if they still exist, and if we can drop this filter. I get these nagging "we own this and should learn what's going on" thoughts. Also, as this is a public repo, some clarification, if even necessary, will help project confidence in our services.

File: models/anonymous_users.sql (hunk 1 above)
Comment (huttotw) [reply models/anonymous_users.sql]: These are an artifact from the early days when testing our Data Destinations service. This never made it to production so I will remove!

**Review by camphillips22 — APPROVED** [review]
LGTM!

**Pull request merged**